### ISSUE TITLE
 a warning about the same api with different http methods

### ISSUE THREAD

**Issue by gotobcn8** [swaggo/swag#1370]
**Describe the bug**
when I run the command to generate swagger doc, the console output warnings like this:

_**warning: route POST /api/clusters/nodes/tablespaces/shrink is declared multiple times**_

**To Reproduce**
1. run cmd: _**swag init --output ./docs -g ./cmd/cod-domor/cod-domor.go --parseDependency --parseDepth=2**_

**Expected behavior**
**_/api/clusters/nodes/tablespaces/shrink_** this api has three http methods (put get post)
so how can I clear this warnings?  as my api is dozens, it would waste time for build swag(output interaction too much).

**Screenshots**

**Your swag version**
swag version v1.7.6

**Your go version**
go version go1.16.6 linux/amd64

**Desktop (please complete the following information):**
 - OS: [e.g. centos7.6]

**Additional context**
not really a bug, just too much, plz can you fix it?

**Comment by gotobcn8** [comment]
I found this not only limits in same api, it also arise in sole api.

**Comment by tmkhoa-dev** [comment]
yeah, I'm also facing with this issue, hope it was be fixed soon.
